nodeJS/testing_express/testing_database_operations.md

```diff
@@ -1,86 +1,70 @@
+- Create and use a separate database for integration testing.
 
-### But do you even need to test that?
+### Unit tests - do you even need to?
 
-Before diving in, you might want to consider whether or not the database operations you're testing even need to be tested in the first place. If you are reading and writing straight from a database using `mongoose` or some other db module, you might not really need to test that code. [Mongoose's GitHub repo's tests directory](https://github.com/Automattic/mongoose/tree/master/test) (and presumably all other popular db modules) already has plenty of tests for all of its actions, so if you are just serving up a JSON API and all you're doing is leveraging functions from another module then those operations are already covered.
+Before diving in, you might want to consider whether or not the database operations you're testing even need to be tested in the first place. If you are reading and writing straight from a database using `pg` or some other db module, you might not really need to test that code. [pg's GitHub repo's tests directory](https://github.com/brianc/node-postgres/tree/master/packages/pg/test) (and presumably all other popular db modules) already has plenty of tests for all of its actions, so if you are just serving up a JSON API and all you're doing is leveraging functions from another module then those operations are already covered.
 
-If your queries are complicated, you might justify adding tests to make sure that you are using them correctly and that the code you have written is doing what you intend it to, and if you are using any of your own code to do some filtering, sorting, or other manipulations of the data you will want to test that as well.  In the case of your own code, however, it would be better if you could pull those things out into their own modules, separate from your database operations so you can test them without messing with the database.
+If your queries are complicated, you might justify adding unit tests to make sure that you are using them correctly and that the code you have written is doing what you intend it to, and if you are using any of your own code to do some filtering, sorting, or other manipulations of the data you will want to test that as well.  In the case of your own code, however, it would be better if you could pull those things out into their own modules, separate from your database operations so you can test them without messing with the database.
 
-### mongodb-memory-server
+### Integration tests
 
-There are cases, however, when you are going to want to test things that touch your database, so this lesson is going to cover how to do that.
+There are cases, however, when you are going to want to test things that touch your database. For example, we performed an integration test on the server in the [testing routes and controllers lesson](./testing_routes_and_controllers.md) using `supertest`.
```

Comment:
I think a full URL would be better here. If dead-set on a relative URL, then replacing `_` for `-` does work, and you can even drop the `.md` as well. But having `_` breaks as our URLs use split with `-`.
```suggestion
There are cases, however, when you are going to want to test things that touch your database. For example, we performed an integration test on the server in the [Testing Routes and Controllers lesson](https://www.theodinproject.com/lessons/nodejs-testing-routes-and-controllers) using `supertest`.
```

Reply:
it should definitely be a full URL, I got lazy lol